core/src/main/resources/lib/form/apply/apply.js

```diff
@@ -1,95 +1,103 @@
+      var f = findAncestor(e.target, "FORM");
 
-    // create a throw-away IFRAME to avoid back button from loading the POST result back
-    id = "iframe" + iota++;
-    target = Element("iframe", {
-      id: id,
-      name: id,
-      style: "height:100%; width:100%",
-    });
-    $(containerId).appendChild(target);
+      // create a throw-away IFRAME to avoid back button from loading the POST result back
+      id = "iframe" + iota++;
+      target = Element("iframe", {
+        id: id,
+        name: id,
+        style: "height:100%; width:100%",
+      });
+      $(containerId).appendChild(target);
 
-    attachIframeOnload(target, function () {
-      if (target.contentWindow && target.contentWindow.applyCompletionHandler) {
-        // apply-aware server is expected to set this handler
-        target.contentWindow.applyCompletionHandler(window);
-      } else {
-        // otherwise this is possibly an error from the server, so we need to render the whole content.
-        var doc = target.contentDocument || target.contentWindow.document;
-        var error = doc.getElementById("error-description");
-        var r = YAHOO.util.Dom.getClientRegion();
-        var contentHeight = r.height / 5;
-        var contentWidth = r.width / 2;
-        if (!error) {
-          // fallback if it's not a regular error dialog from oops.jelly: use the entire body
-          error = Element("div", { id: "error-description" });
-          error.appendChild(doc.getElementsByTagName("body")[0]);
-          contentHeight = (r.height * 3) / 4;
-          contentWidth = (r.width * 3) / 4;
-        }
+      attachIframeOnload(target, function () {
+        if (
+          target.contentWindow &&
+          target.contentWindow.applyCompletionHandler
+        ) {
+          // apply-aware server is expected to set this handler
+          target.contentWindow.applyCompletionHandler(window);
+        } else {
+          // otherwise this is possibly an error from the server, so we need to render the whole content.
+          var doc = target.contentDocument || target.contentWindow.document;
+          var error = doc.getElementById("error-description");
+          var r = YAHOO.util.Dom.getClientRegion();
+          var contentHeight = r.height / 5;
+          var contentWidth = r.width / 2;
+          if (!error) {
+            // fallback if it's not a regular error dialog from oops.jelly: use the entire body
+            error = Element("div", { id: "error-description" });
+            error.appendChild(doc.getElementsByTagName("body")[0]);
+            contentHeight = (r.height * 3) / 4;
+            contentWidth = (r.width * 3) / 4;
+          }
 
-        let oldError = $("error-description");
-        if (oldError) {
-          // Remove old error if there is any
-          $(containerId).removeChild(oldError);
-        }
+          let oldError = $("error-description");
+          if ((oldError = $("error-description"))) {
```

Comment:
What is this magic? 🤔

Reply:
Yeah I'm confused by it too - it was there before it's just been linted/reindented.

Reply:
I see indentation change, but condition inside the `if` was different
![image](https://user-images.githubusercontent.com/91559310/194856184-992fc1fd-6fb3-4e25-ad07-bc99113b5829.png)

Reply:
```suggestion
          if (oldError) {
```